### PR TITLE
Update Abstract

### PR DESCRIPTION
**Pull request by David-Chadwick**
This updates the Abstract to be consistent with the Introduction

***
<a href="https://pr-preview.s3.amazonaws.com/David-Chadwick/vc-jose-cose/pull/213.html" title="Last updated on Jan 6, 2024, 2:22 PM UTC (9df21fc)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/213/e4eae17...David-Chadwick:9df21fc.html" title="Last updated on Jan 6, 2024, 2:22 PM UTC (9df21fc)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -134,7 +134,7 @@
     <p>
       This specification defines how to secure credentials and presentations
       conforming to the Verifiable Credential data model [[VC-DATA-MODEL-2.0]]
-      with JSON Object Signing and Encryption (<a href="https://datatracker.ietf.org/wg/jose/about/">JOSE</a>)
+      by using SD-JWT (<a href="https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-07.html">JOSE</a>)
```
Comment (TallTed) [inline index.html:137]:
```suggestion
      by using Selective Disclosure for JWTs (<a href="https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/07/">SD-JWT</a>)
```

**Comment by selfissued** [comment]
If you'd like the working group to discuss this topic, please open an issue for the discussion.  Before a PR is created, we normally first obtain working group consensus on any change to be made.  Please close this PR until such time as there is working group consensus to make a change.  Thank you.

**Comment by David-Chadwick** [comment]
the issue already exists. It is #201

**Comment by decentralgabe** [comment]
This should be closed until we have consensus in an issue.

**Comment by decentralgabe** [comment]
Closing, as we're headed in the direction of including all of JOSE.

**Pull request closed without merging**